utils/perl5lib/Module/ModuleLoader.pm

```diff
@@ -289,34 +290,34 @@ sub findModulesForCase()
     }
     my $parser = XML::LibXML->new(no_blanks => 1);
     my $casemoduleparser = $parser->parse_file($self->{machspecificfile});
-    my @allmodulenodes = $casemoduleparser->findnodes("/machine[\@MACH=\'$machine\']/module_system/modules");
+    my @allmodulenodes = $casemoduleparser->findnodes("//module_system/modules");
```

Comment:
More perl changes in code that has been ported to python. This makes me nervous that one or the other is not getting the updates it needs.